django_cloud_deploy/crash_handling/__init__.py

```diff
@@ -44,12 +51,12 @@ def handle_crash(err: Exception, command: str,
         console: Object to use for user I/O.
     """
 
-    # TODO: Only handle crashes caused by our code, not user's code.
+    # Only handle crashes caused by our code, not user's code.
     # When deploying, our tool will run the code of user's Django project.
-    # If user's code has a bug, then the traceback will contain the directory
-    # path of their Django project. We will put traceback in a public issue,
-    # but user's might not want to put these kind of information to public,
-    # so we plan to only handle crashes caused by our code.
+    # If user's code has a bug, then an UserError will be raised. In this case,
+    # we do not want users to create a Github issue.
+    if type(err).__name__ in _KNOWN_EXCEPTIONS:
+        raise err
```

Comment:
Would it make sense to do this?

```suggestion
        raise ex.__cause__
```

Reply:
Done.